pkg/ddevapp/config.go

```diff
@@ -739,6 +739,17 @@ func (app *DdevApp) CheckCustomConfig() {
 		}
 	}
 
+	configPath := filepath.Join(ddevDir, "")
+	if _, err := os.Stat(configPath); err == nil {
+		entrypointFiles, err := filepath.Glob(filepath.Join(configPath, "config.*.y*ml"))
+		util.CheckErr(err)
+		if len(entrypointFiles) > 0 {
+			printableFiles, _ := util.ArrayToReadableOutput(entrypointFiles)
+			util.Warning("Using custom config.*.y*ml configuration: %v", printableFiles)
```

Comment:
Let's use `Using custom config.*.yaml` in the output even if it's not true (because we use `config.*.yaml` everywhere in the docs).

```suggestion
		configFiles, err := filepath.Glob(filepath.Join(configPath, "config.*.y*ml"))
		util.CheckErr(err)
		if len(configFiles) > 0 {
			printableFiles, _ := util.ArrayToReadableOutput(configFiles)
			util.Warning("Using custom config.*.yaml configuration: %v", printableFiles)
```